openshift/consumer-template.yaml

```diff
@@ -95,6 +100,8 @@ objects:
                   value: ${{DEBUG_LEVEL}}
                 - name: APP_SCRIPT
                   value: "app.sh"
+                - name: THOTH_ENVIRONMENT
+                  value: "${THOTH_ENVIRONMENT}"
```

Comment:
We have ``THOTH_DEPLOYMENT_NAME`` that has slightly different semantics but captures test/stage/prod with cluster name. Would it be more suitable?

Reply:
Yes in this case that is reusable actually, thanks!